src/Files/MultilingualResources/Files.id-ID.xlf

```diff
@@ -3369,155 +3369,155 @@ Kami menggunakan App Center untuk melacak penggunaan aplikasi, menemukan bug, da
         </trans-unit>
         <trans-unit id="ThirdPartyLicenses" translate="yes" xml:space="preserve">
           <source>Third party licenses</source>
-          <target state="new">Third party licenses</target>
+          <target state="new">Lisensi Pihak ke tiga</target>
```

Comment:
Please change the `state="new"` to `state="translated"` and in other places too!